magma/db/migrations/011_add_gnomon_tables.rb

```diff
@@ -0,0 +1,27 @@
+Sequel.migration do
+  change do
+    create_table(:rules) do
+      primary_key :id
+
+      String :project_name, null: false
+      Integer :version_number, null: false
+      column :config, :json, null: false
+      DateTime :created_at, null: false
+
+      index [:project_name, :version_number], unique: true
+    end
+
+    create_table(:identifiers) do
+      primary_key :id
+
+      String :project_name, null: false
```

Comment:
Would it be helpful in any way to also record which `model` an identifier belongs to? Thinking it may save some sort of computation when searching for identifiers perhaps, being able to construct the rule directly instead of having to iterate through all rules to find the matching one?